charts/datadog/values.yaml

```
@@ -87,6 +87,22 @@ datadog:
     ## Migration guide: https://docs.datadoghq.com/integrations/kubernetes_state_core/?tab=helm#migration-from-kubernetes_state-to-kubernetes_state_core
     ignoreLegacyKSMCheck: true
 
+    # datadog.kubeStateMetricsCore.labelsAsTags --
+    ## Extra labels to collect from resources and to turn into datadog tag.
```

```suggestion
    # datadog.kubeStateMetricsCore.labelsAsTags -- Extra labels to collect from resources and to turn into datadog tag.
```
to make the comment appear in the readme

---

👍🏻 Good idea.
Done !